### PR TITLE
Bugfix/168_jsonUtil

### PR DESCRIPTION
**Pull request by Mortadha-abderrhim**
Added Audio, Video, and Stats to the experience interpreted from the JSON.
Added the other parameters for the constructors (type, status, ...)

### CODE REVIEW

File: app/src/main/java/com/pindex/main/utils/JsonUtils.kt [hunk 1]
```diff
@@ -106,6 +103,47 @@ object JsonUtils {
...
+            type = json?.get("type")?.jsonPrimitive?.content,
+            title = json?.get("title")?.jsonPrimitive?.content,
+            value = json?.get("value")?.jsonPrimitive?.content
+        )
+    }
+    /**
+     * Creates an [TimetableBlockDto.HourEntryDto] from [JsonObject]
+     */
+    /*fun hourEntryFromJson(json: JsonObject?):TimetableBlockDto.HourEntryDto{
+        return TimetableBlockDto.HourEntryDto(
```
Comment (hadiyahya11) [inline app/src/main/java/com/pindex/main/utils/JsonUtils.kt:130]: Delete these lignes if unused

**Review by hadiyahya11 — APPROVED** [review]
Nice job !

File: app/src/main/java/com/pindex/main/utils/JsonUtils.kt (hunk 1 above)
Comment (Mortadha-abderrhim) [reply app/src/main/java/com/pindex/main/utils/JsonUtils.kt:130]: They would be helpful for adding another block I didn't manage to do for this sprint

**Review by aouame — APPROVED** [review]
Gj, the code is well organised

**Pull request merged**